nss-day-cohort-39/hello-world-the-globtrotters
# Automating Sweden

### Details
- Sweden cities, landmarks and leaders are now populated through Javascript

### How to test
- Make sure cities, landmarks and leaders are populated through Javascript instead of sweden.html
- Make sure the information doesn't populate onto the wrong pages
